src/store/values/Participants.js

```diff
@@ -25,6 +30,14 @@ const participants = createSlice({
   },
 });
 
+export const getCurrentUserById = createSelector(
```

Comment:
Maybe rename it to a better meaning, w.g. we do not pass any `id` so we can remove the `byId` postfix.

Probably we could name it as `getCurrentUserFromParticipants` ? 

And I have more general question - is just calling `selectCurrentUser` not enough here? Do we really need to update it with the data from participants ?

Reply:
@Oleksandr1414 I agree